R-package/README.md

```diff
@@ -26,6 +26,13 @@ For users who wants to install online with GPU or want to choose a specific comp
 
 You can perform installation either with **Apple Clang** or **gcc**. In case you prefer **Apple Clang**, you should install **OpenMP** (details for installation can be found in [Installation Guide](https://github.com/Microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#apple-clang)) first and **CMake** version 3.12 or higher is required. In case you prefer **gcc**, you need to install it (details for installation can be found in [Installation Guide](https://github.com/Microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#gcc)) and set some environment variables to tell R to use `gcc` and `g++`. If you install these from Homebrew, your versions of `g++` and `gcc` are most likely in `/usr/local/bin`, as shown below.
 
+#### Linux OS Preparation
```

Comment:
The steps are not for Linux OS but are for Ubuntu in your case. It will not work on other Linux OSes.

Reply:
Agree with @Laurae2 , `apt` package manager isn't going to exist on all OSes